addons/binding/org.openhab.binding.kostalinverter/README.md

```diff
@@ -4,6 +4,8 @@ Scrapes the web interface of the inverter for the metrics of the supported chann
 
 ![Kostal Pico](doc/kostalpico.jpg)
 
+![Kostal PLENTICORE / PIKI IQ](doc/PLENTICORE.tif)
```

Comment:
Can you make it a jpg. It's now a rather large image. (also maybe use lowercase for filename).

Reply:
done :)

Reply:
Hmm. I would have expected the image to be smaller,  but it's not much smaller than the tif :confused:. So I imported it in GIMP, were it complains about color space and exported it. This results in a ~45kb image. Can you see if you can reduce the size of the image?

Reply:
14,6 kB now :)